### DISCUSSION TITLE
GUI for the CFD solver SU2

### DISCUSSION THREAD

**Discussion by bigfooted** [Kitware/trame#437] [Show and tell]
Hi all,
I gave a short presentation on my progress in building a GUI for the CFD solver SU2 last October. For those who are interested, the presentation with video is here, day 1 at 15:00:15:20
[su2 conference 2023](https://su2foundation.org/su2conference2023/)

**Comment by jourdain** [comment]
Thanks for sharing and feel free to do a PR adding your tool to the community section [here](https://kitware.github.io/trame/examples/apps/community.html) / [markdown](https://github.com/Kitware/trame/blob/master/docs/vitepress/examples/apps/community.md).
